### DISCUSSION TITLE
On click for outside of image

### DISCUSSION THREAD

**Discussion by zax71** [igordanchenko/yet-another-react-lightbox#95] [Q&A] [answered]
How would I close the lighthouse modal when a user clicks outside the image? I've found `on={{click: () => setOpen(false)}}` but that only seems to be for inside the image, not outside.

**Comment by igordanchenko** [comment, marked as answer]
I believe you are looking for [closeOnBackdropClick](https://yet-another-react-lightbox.com/documentation) option.
